datafusion/sqllogictest/test_files/arrow_files.slt

```diff
@@ -43,6 +43,10 @@ SELECT * FROM arrow_simple
 3 baz false
 4 NULL true
 
+# url table is only supported by DynamicFileCatalog
```

Comment:
I think adding context here about *why* this is an important test would make it easier to understand for future readers

```suggestion
# Ensure that local files can not be read by default (a potential security issue)
# (url table is only supported when DynamicFileCatalog is enabled)
```